docs/howto/deploy/gcp.md

```diff
@@ -11,9 +11,15 @@ next:  ["Import data into your installation", "/howto/import.html"]
 
 # Deploy lakeFS on GCP
 
-These instructions are for a self-managed deployment of lakeFS on GCP. <br/> 
-For a hosted lakeFS service with guaranteed SLAs, try [lakeFS Cloud](https://lakefs.cloud/).
-{: .note }
+
+{: .tip }
+> The instructions given here are for a self-managed deployment of lakeFS on GCP. 
+> 
+> For a hosted lakeFS service with guaranteed SLAs, try [lakeFS Cloud](https://lakefs.cloud)
```

Comment:
For GCP, we might want to send the users to the documentation link since it is not yet self managed and requires us to set it up for the user.

Reply:
I think we should either leave this as-is, or say "for lakeFS Cloud on GCP please contact us" which is a pattern we use elsewhere. 
A link back to the docs would be circular. 
Alternatively we just remove this callout for GCP. 
WDYT?

Reply:
I'm fine with either

Reply:
updated.